### PR TITLE
Correct some wrong info.

### PR DESCRIPTION
**Pull request by BeastGamer81**
Corrected information - 
- Location
- Aliases
- Truth about german location

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,15 +89,15 @@ <h1 class="text-white font-rajdhani text-left text-xl md:text-3xl">
...
-                            He is commonly referred to by various aliases such as: shadowbeast, nightshade, darknight, shadow skid, among others. Despite being a 15 year old kid, he portrays himself as an adult in his early twenties. However, his real name is Faiz Patel and he lives in Kolkata, India.
+                            He is commonly referred to by various aliases such as: BeastGamer81, darknight, darknight81, among others. Despite being a 15 year old kid, he portrays himself as an adult in his early twenties. However, his real name is Faiz Patel and he lives in Maharashtra, India.
                         </p>
                     </div>
                     <div class="flex flex-col">
                         <h1 class="text-white font-rajdhani text-left text-xl md:text-3xl">
                            2. What has he done?
                         </h1>
                         <p>
-                            His criminal activities encompass a wide array of offenses, including exploiting trial periods without authorization, fabricating counterfeit credit cards, manipulating server providers, circumventing FOSS License regulations, and numerous others.
```
Comment (recabasic) [inline index.html:100]: dlgtm

File: index.html [hunk 2]
```diff
@@ -125,10 +125,7 @@ <h1 class="text-white font-rajdhani text-left text-xl md:text-3xl">
                                     iii. Perpetration of fraud against Mr. Piecer's Datalix account.
                                 </li>
                                 <li>
-                                    iv. Baseless accusations directed towards HolaClient and its personnel.
```
Comment (recabasic) [inline index.html:128]: Nuh uh, You did it

File: index.html [hunk 3]
```diff
@@ -146,7 +143,8 @@ <h1 class="pt-4 text-gray-200 font-rajdhani font-bold text-center text-3xl md:te
...
...ordable, reliable, and reputable providers worldwide. I vividly recall the incessant barrage of messages he sent me, seeking assistance. It appears he's still soliciting help from others. I understand the need for guidance when learning something new, but I would have gladly assisted him if he had approached me with genuine curiosity about the subject matter. Instead, he opted to develop software in the same industry as mine. Despite this, I continued to aid him until he attempted to belittle me. However, such behavior is unwarranted from someone of his age. I have never sought assistance from anyone regarding programming. The evidence, in the form of screenshots, is provided below. I've compressed them into base64 format to conserve bandwidth and enhance loading speed. Thank you for reading through this rather lengthy discourse. <span class="font-bold">Update1: HolaClient has revoked the usage access of the Zenith and Purplenights themes in KlovitClient, including its login page. Using it further will result in DMCA takedown.</span><span class="font-bold">Fresh Update: The Germany location wasn't actually German it was French and was renamed to France after realiszing that.</span>
```
Comment (recabasic) [inline index.html:146]: I don't think so.

**Review by recabasic — CHANGES_REQUESTED** [review]
Will be approved after requested changes are cleared.

File: index.html (hunk 1 above)
Comment (BeastGamer81) [reply index.html:100]: I never fabricated counterfeit credit cards and trials

Comment (BeastGamer81) [reply index.html:100]: however if you want to revert it, you can.

File: index.html (hunk 3 above)
Comment (BeastGamer81) [reply index.html:146]: You can ask Zewoo and the owners of AnxiousPlanet.cloud

File: index.html (hunk 2 above)
Comment (BeastGamer81) [reply index.html:128]: Well, not against HC only Demon and I regret it.

**Comment by recabasic** [comment]
Doesn't look good to me, tried to evade information. Only one change will be accepted (location).

**Comment by BeastGamer81** [comment]
> Doesn't look good to me, tried to evade information. Only one change will be accepted (location).

You have no proofs about the German location, our IPv6 is German and that's it.

**Pull request closed without merging**